### PR TITLE
Refactor: Price component

### PR DESCRIPTION
**Pull request by RobinCsl**

### CODE REVIEW

File: app/hotels/src/map/allHotels/PriceMarker.js [hunk 1]
```diff
@@ -27,15 +27,7 @@ const PriceMarker = (props: Props) => {
   return (
     <View>
       <View style={bubbleStyles}>
-        {price &&
```
Comment (tbergq) [inline app/hotels/src/map/allHotels/PriceMarker.js]: I would prefer that this check is price != null &&

File: app/hotels/src/map/hotelDetailPreview/HotelDetailPreview.js [hunk 2]
```diff
@@ -53,13 +53,7 @@ export default function HotelDetailPreview(props: Props) {
...
               <View>
                 {stars > 0 && <Stars rating={stars} style={styles.stars} />}
-                {price &&
-                  price.currency != null &&
-                  price.amount != null && (
-                    <Text style={styles.price}>
-                      <Price currency={price.currency} amount={price.amount} />
-                    </Text>
-                  )}
+                {price && <Price price={price} style={styles.price} />}
```
Comment (tbergq) [inline app/hotels/src/map/hotelDetailPreview/HotelDetailPreview.js]: same here

**Comment by RobinCsl** [comment]
@tbergq  🚀

**Comment by RobinCsl** [comment]
Fixes #1208

**Pull request merged**